admin/spec/models/check-window.spec.js

```diff
@@ -35,7 +50,7 @@ describe('check-window schema', function () {
     })
   })
 
-  it('requires a adminStartDate property', function (done) {
+  it('should require a adminStartDate property', function (done) {
```

Comment:
again, i would assert the error details

`expect(error.message).toBe('admin start date required')`